src/main/resources/assets/admin/common/js/ui/dialog/ConfirmationDialog.ts

```diff
@@ -67,8 +65,15 @@ export class ConfirmationDialog
         super.initElements();
 
         this.questionEl = new H6El('question');
-        this.noAction = new Action(i18n('action.no'), 'esc');
-        this.yesAction = new Action(i18n('action.yes'));
+
+        this.noAction = new Action(i18n('action.no'), i18n('action.no').slice(0, 1).toLowerCase());
+        this.noAction.setMnemonic(i18n('action.no').slice(0, 1).toLowerCase());
+
+        this.yesAction = new Action(i18n('action.yes'), i18n('action.yes').slice(0, 1).toLowerCase());
+        this.yesAction.setMnemonic(i18n('action.yes').slice(0, 1).toLowerCase());
```

Comment:
Can we avoid calling `i18n` twice for the same key? This will actually browse all resources every time it's called...

Reply:
done